packages/components/src/overlays/Popover/src/Popover.module.css

```diff
@@ -0,0 +1,99 @@
+:where(.hop-Popover *) {
+    font-family: var(--font-family);
+    font-size: var(--font-size);
+    font-weight: var(--font-weight);
+    line-height: var(--line-height);
+    color: var(--color);
```

Comment:
setting color here is related to the issue you're having with the link not having the right color.
I now see that setting color on every single child is most likely quite dangerous. I'd prioritize setting the link variant, and do color a different way int he css.